syntax="proto2";
option cc_generic_services = true;

message EchoRequest {
  required string msg = 1;
}

message EchoResponse {
  required string echo_msg = 1;
}

service EchoService {
  rpc Echo(EchoRequest) returns (EchoResponse);
}